applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -162,6 +162,28 @@ BUILD_ASSERT(CONFIG_BT_ASCS_ASE_SRC_COUNT <= 1,
 	     "CIS headset only supports one source stream for now");
 #endif /* (CONFIG_BT_AUDIO_TX) */
 
+static int codec_freq_get_hz(const struct bt_audio_codec_cfg *codec)
+{
+	enum bt_audio_codec_config_freq freq_id = bt_audio_codec_cfg_get_freq(codec);
+
+	if (freq_id < 0) {
+		return freq_id;
+	}
+
+	return (bt_audio_codec_cfg_freq_to_freq_hz(freq_id));
+}
+
+static int codec_duration_us(const struct bt_audio_codec_cfg *codec)
+{
+	enum bt_audio_codec_config_frame_dur frame_dur = bt_audio_codec_cfg_get_frame_dur(codec);
```

Comment:
Have a look at other places where `bt_audio_codec_cfg_get_freq` and 'bt_audio_codec_cfg_get_frame_dur' are used. Seems the commonly return to a type `int` and not an enum since they can contain negative values.